Tasks/PublishTestResults/publishtestresults.ts

```diff
@@ -30,10 +30,11 @@ else {
   var matchingTestResultsFiles = [testResultsFiles];
 }
 
-if(!matchingTestResultsFiles) {
+if(!matchingTestResultsFiles || matchingTestResultsFiles.length == 0) {
```

Comment:
[http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons](url)